shell/platform/darwin/ios/framework/Source/FlutterPlatformViews_Internal.h

```diff
@@ -261,7 +261,7 @@ class FlutterPlatformViewsController {
                    const std::shared_ptr<IOSContext>& ios_context,
                    std::unique_ptr<SurfaceFrame> frame);
 
-  void OnMethodCall(FlutterMethodCall* call, FlutterResult& result);
+  void OnMethodCall(FlutterMethodCall* call, FlutterResult result);
```

Comment:
FlutterResult& result in ARC doesn't work. And I don't think having a pointer to the objc block as the parameter made sense at first place? The objc block is already a pointer. 
I might be wrong.